bin/node/runtime/src/weights/pallet_balances.rs

```diff
@@ -0,0 +1,29 @@
+use frame_support::weights::{Weight, constants::RocksDbWeight as DbWeight};
+pub struct WeightInfo;
+impl pallet_balances::WeightInfo for WeightInfo {
+	fn transfer() -> Weight {
+		(65949000 as Weight)
```

Comment:
Are the casts to `Weight`, here and below, needed?

Reply:
The cast to weight comes from the automation pipeline. I add it everywhere to ensure types are all defined and there wont be compile errors.

Is there something better I can do?

Reply:
I suspected that this file was auto-generated but wasn't sure, I guess we can keep the casts then to make it unambiguous.